metadata_service/proxy/atlas_proxy.py

```diff
@@ -706,6 +734,47 @@ def get_table_by_user_relation(self, *, user_email: str, relation_type: UserReso
                 cluster=res['cluster'],
                 schema=res['db'],
                 name=res['table']))
+        return results
+
+    def _get_resource_owned_by_user(self, user_id: str, resource_type: str) \
+            -> List[Union[PopularTable, DashboardSummary, Any]]:
+        """
+        ToDo (Verdan): Dashboard still needs to be implemented.
+        Helper function to get the resource, table, dashboard etc owned by a user.
+        :param user_id: User ID of a user
+        :param resource_type: Type of a resource that returns, could be table, dashboard etc.
+        :return: A list of PopularTable, DashboardSummary or any other resource.
+        """
+        owned_by_resources = list()
+        user_entity = self._driver.entity_unique_attribute(self.USER_TYPE, qualifiedName=user_id).entity
+
+        if not user_entity:
+            LOGGER.exception(f'User ({user_id}) not found in Atlas')
+            raise NotFoundException(f'User {user_id} not found.')
+
+        resource_guids = list()
+        for item in user_entity[self.REL_ATTRS_KEY].get('ownerOf') or list():
+            if (item['entityStatus'] == Status.ACTIVE and
+                    item['relationshipStatus'] == Status.ACTIVE and
+                    item['typeName'] == resource_type):
```

Comment:
can those conditions be aligned for readability?

Reply:
auto indent 😉